test/auth_cluster/test_raft_service_levels.py

```diff
@@ -136,6 +137,10 @@ async def test_service_levels_work_during_recovery(manager: ManagerClient):
     assert sl_v1 not in [sl.service_level for sl in recovery_result]
     assert set([sl.service_level for sl in recovery_result]) == set(sls)
 
+    logging.info("Checking changes to service levels are forbidden during recovery")
+    with pytest.raises(InvalidRequest, match="The cluster is in recovery mode. Changes to service levels are not allowed."):
+        await cql.run_async(f"CREATE SERVICE LEVEL sl_{unique_name()    }")
```

Comment:
It would be nice to check other possible operations on service levels as well - altering, dropping, attaching, detaching.

Reply:
attaching/detaching a service level is part of auth, which also prints the same ugly message in recovery

Reply:
> However attaching/detaching a service level is part of auth, which also prints the same ugly message in recovery

Ok, no need to increase the scope of the PR to cover them, then. I'll open an issue about the ugly error messages in auth, this can be handled there.